### PR TITLE
Umbrella for release 2023-H1

### PR DESCRIPTION
**Pull request by bowni**
## Changes in elements and items

- [x] Issue #196 on logging of inferences/predictions: as a new element 5.6
- [x] Issue #197 on enhancing element 6.1 with aggegated carbon footprint of AI activities, and/or a SCAP
- [x] Broaden language of element 6.1 to energy consumption measurement (issue #197)

## Changes in resources

- [x] Add https://arxiv.org/ftp/arxiv/papers/2104/2104.10350.pdf to element 6.1 (issue #205)
- [x] Add https://arxiv.org/abs/2111.00364 to element 6.1 or to section 6 directly? (issue #206)
- [X] Add Quantmetry's blog article on frugality (issue #202)
- [x] Issue #203 to introduce a first reference of compliance of foundational models / LLMs with AI Act

## Misc. changes

None

## Comments

Topics for the next release (2023-H2):

- [ ] New evaluation element on frugality and model pruning/compression (cf. issue #213)

### CODE REVIEW

**Comment by bowni** [comment]
@ClementMayer @RomainGoussault @nicolas-landel @SaboniAmine @meuce @celinejacques can I count on you for reviewing this update? 😎

File: referentiel_evaluation.md [hunk 1]
```diff
@@ -1060,7 +1090,10 @@ Il est important de s'interroger et de conscientiser les coûts environnementaux
 
 - (Software & Tools) *[ML Impact Calculator](https://mlco2.github.io/impact/)*
 - (Software & Tools) *[Code Carbon](https://codecarbon.io/)*: librairie Python permettant d'évaluer le coût carbone de l'exécution d'un script
+- (Web article) *[La frugalité, ou comment empêcher l’IA de franchir les limites](https://www.quantmetry.com/blog/ia-confiance-frugalite/)*, Geoffray Brelurut (Quantmetry), Juin 2023
+- (Academic paper) *[Carbon Emissions and Large Neural Network Training](https://arxiv.org/ftp/arxiv/papers/2104/2104.10350.pdf)*, David Patterson, Joseph Gonzalez, Quoc Le, Chen Liang, Lluis-Miquel Munguia, Daniel Rothchild, David So, Maud Texier, Jeff Dean, 2021. Extrait : *Remarkably, the choice of DNN, datacenter, and processor can reduce the carbon footprint up to ~100-1000X*
```
Comment (SaboniAmine) [inline referentiel_evaluation.md:1094]: Référence suggérée : 
(Academic paper) *[Estimating the carbon footprint of Bloom, a 176B parameter language model](https://arxiv.org/pdf/2211.02001.pdf)*, Alexandra Sasha Luccioni, Sylvain Viguier, Anne-Laure Ligozat, 2022. Extract : *While we will predominantly focus on model training, we will also take into account the emissions produced by manufacturing the computing equipment used for running the training, the energy-based operational emissions, as well as the carbon footprint of model deployment and inference*

**Review by SaboniAmine — COMMENTED** [review]
Merci pour cette mise à jour!
Un petit commentaire sur l'historisation des prédictions faites par les modèles & les aspects de confidentialité, ainsi qu'une suggestion de référence à ajouter côté estimation de l'impact environnemental.

File: assessment_framework_eng.md [hunk 2]
```diff
@@ -1016,6 +1023,27 @@ Using automatic systems based on models whose rules have been "learned" (and not
...
+
+- [ ] 5.6.a Our organisation does not use AI models on its own behalf or on behalf of its clients, and does not provide its clients with applications based on AI models | _(Concerned / Not concerned)_
+- [ ] 5.6.b Logging predictions from AI models used in production is not yet systematically implemented
+- [ ] 5.6.c We systematically log all predictions from AI models used in production (coupled with the input data and the associated models references)
+
+<details>
+<summary>Expl5.6 :</summary>
+
+Using automatic systems based on AI models whose rules have been learned questions the way organisations design and operate their products and services. It is important to preserve the responsiveness and resilience of organisations using those AI models, particularly in dealing with situations where AI models have led to an undesirable outcome for the organisation or its stakeholders. To that end, logging predictions from AI models used in production  (coupled with the input data and the associated models references) is key to enable ex-post auditability on concrete use cases.
```
[the remark this answers is not in the document]
Comment (bowni) [reply assessment_framework_eng.md:1043]: It is a very valid concern. Logging predictions for auditability and conflict resolution purposes is too. I believe both should be combined by demanding a strict compliance to GDPR requirements in the way predictions logging is put in place.

**Comment by nicolas-landel** [comment]
I don't have much to say about the content as I don't keep a close eye on these topics anymore.

Concerning the form, everything looks good to me :)

**Review by nicolas-landel — DISMISSED** [review]
I don't have much to say about the content as I don't keep a close eye on these topics anymore.

Concerning the form, everything looks good to me :)

**Comment by bowni** [comment]
@SaboniAmine additions made to cover your comments!

**Review by SaboniAmine — APPROVED** [review]

**Review by ClementMayer — APPROVED** [review]
Ok pour moi. 
Sur le deuxième semestre il faudra sans doute lancer un chantier generative AI et ses spécificités (droit d'auteurs, interprétability (?), lutte contre les halucinations et capacité à identifier les sources, sans doute aussi quelques sujets spécifiques sur la taille des modèles)

**Review by celinejacques — APPROVED** [review]
Only one typo spotted
Otherwise ready to merge

**Review by RomainGoussault — APPROVED** [review]
Looks good to me! Thanks @bowni!

File: assessment_framework_eng.md [hunk 3]
```diff
@@ -1029,24 +1057,26 @@ The implementation of an automatic system based on an AI model can generate nega
...
 
-- [ ] 6.1.a At this stage we have not looked at the CO2 impact of our data science activity or our AI models
-- [ ] 6.1.b We have developed indicators that define what we want to measure regarding the CO2 impact of our data science activity or our models
+- [ ] 6.1.a At this stage we have not studied specifically the environmental impact of our data science activity or our AI models
+- [ ] 6.1.b We have developed indicators that define what we want to measure regarding the energy consumption and the carbon footprint of our data science activity or our models
 - [ ] 6.1.c We measure our indicators regularly
 - [ ] 6.1.d We include their measurements in the model identity cards
 - [ ] 6.1.e Monitoring our indicators on a regular basis is a formalised and controlled process, from which we define and drive improvement objectives
-- [ ] 6.1.f The CO2 impact of our data science activity or our models is made transparent to our counterparts and the general public
+- [ ] 6.1.f We consolidate an aggregated view of the energy consumtion and carbon footprint of our data science activities
```
[the remark this answers is not in the document]
Comment (bowni) [reply assessment_framework_eng.md:1072]: Ok thank you, well spotted! Will fix it after the main merge, in a hotfix branch 👌

**Pull request merged**